apps/src/music/blockly/blocks/samples.js

```diff
@@ -42,12 +36,7 @@ export const playSound = {
     nextStatement: null,
     colour: 230,
     tooltip: 'play sound',
-    helpUrl: '',
-    extensions: [
-      'dynamic_menu_extension',
-      'preview_extension',
-      'clear_preview_on_change_extension'
```

Comment:
We can also remove these extension definitions in MusicView right?

Reply:
Good call.